### ISSUE TITLE
h1 + p from embedded css only taken after changing font size

### ISSUE THREAD

**Issue by maniacgit** [koreader/koreader#1381]
I'm using h1 for starting a new chapter in my epubs. And I've set a general indent for all p. But the first paragraph after a headline should not be indented.

For this I've created the following entries in the embedded css in the epub.

p {
  text-indent:0.5em;
  margin-left:0;
  margin-right:0;
  text-align:justify;
  margin-top:0;
  margin-bottom:0.2em
}
h1 + p { text-indent:0; }

But if I open a book the first paragraph after the h1 has still the indent. If I change the font size everything works like expected, even if I change the font size back to the former value.

I'm using a Kobo H2O with v2014.11-15-gfc0e24a

**Comment by Frenzie** [comment]
Huh, I created this quick test document based on your info and it crashes my dev instance.

```
<!DOCTYPE html>
<title>bla</title>
<style>
p {
text-indent:0.5em;
margin-left:0;
margin-right:0;
text-align:justify;
margin-top:0;
margin-bottom:0.2em
}
h1 + p { text-indent:0; }
</style>
<h1>bla</h1>
<p>blabla</p>
```

**Comment by a-kohout** [comment]
Related: #2845 
Testcase is reproducible on v2015.11-1024-g3b813c7.
Maybe crengine does support adjacent siblings but this weird bug is getting in the way.

**Comment by poire-z** [comment]
Original issue fixed by #3922

**Comment by Frenzie** [comment]
PS https://github.com/koreader/crengine/issues/177 is with #3922 (and #3928) included but seems to behave rather similar.
